ccd-definition/ComplexTypes/PayingMoneyAsAResultOfCourtOrder.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "ID": "PayingMoneyAsAResultOfCourtOrder",
+    "ListElementCode": "payingMoneyDetailsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Is ${respondent1.partyName} paying money as a result of any court order? \n",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "PayingMoneyAsAResultOfCourtOrder",
+    "ListElementCode": "payingDetailsRequired",
+    "FieldType": "YesOrNo",
+    "ElementLabel": " ",
```

Comment:
If you set the previous Label's ElementLabel here as this ElementLabel, does the result look the same? If so, you can save the payingMoneyDetailsLabel element